LDAPCollector.class.inc.php

```diff
@@ -154,15 +154,40 @@ public function Search($sDN, $sFilter, $aAttributes = array('*'))
         if ($this->Connect())
         {
             Utils::Log(LOG_DEBUG, "ldap_search('$sDN', '$sFilter', ['".implode("', '", $aAttributes)."'])...");
+
+	$cookie = '';
+	$pageSize = 500;
+	$firstPass = true;
+	do {
+	    ldap_control_paged_result($this->rConnection, $pageSize, true, $cookie);
```

Comment:
This is deprecated :/
See https://www.php.net/manual/en/function.ldap-control-paged-result : 
> This function has been DEPRECATED as of PHP 7.4.0, and REMOVED as of PHP 8.0.0. Instead the controls parameter of ldap_search() should be used. See also LDAP Controls for details.